tools/build.ts

```diff
@@ -138,75 +110,10 @@ if (!buildWasm) {
     fs.writeFileSync(OUT_WASM_JS, `;throw new Error("please build WebAssembly before use wasm backend.");`);
   }
 } else {
-  // Step 1: emsdk install (if needed)
-  npmlog.info('Build.Wasm', '(1/4) Setting up emsdk...');
-  if (!fs.existsSync(DEPS_EMSDK_EMSCRIPTEN)) {
-    npmlog.info('Build.Wasm', 'Installing emsdk...');
-    const install = spawnSync(`${EMSDK_BIN} install latest`, {shell: true, stdio: 'inherit', cwd: DEPS_EMSDK});
-    if (install.status !== 0) {
-      if (install.error) {
-        console.error(install.error);
-      }
-      process.exit(install.status === null ? undefined : install.status);
-    }
-    npmlog.info('Build.Wasm', 'Installing emsdk... DONE');
-
-    npmlog.info('Build.Wasm', 'Activating emsdk...');
-    const activate = spawnSync(`${EMSDK_BIN} activate latest`, {shell: true, stdio: 'inherit', cwd: DEPS_EMSDK});
-    if (activate.status !== 0) {
-      if (activate.error) {
-        console.error(activate.error);
-      }
-      process.exit(activate.status === null ? undefined : activate.status);
-    }
-    npmlog.info('Build.Wasm', 'Activating emsdk... DONE');
-  }
-  npmlog.info('Build.Wasm', '(1/4) Setting up emsdk... DONE');
-
-  // Step 2: Find path to emcc
-  npmlog.info('Build.Wasm', '(2/4) Find path to emcc...');
-  let emcc = globby.sync('./**/emcc', {cwd: DEPS_EMSDK_EMSCRIPTEN})[0];
-  if (!emcc) {
-    npmlog.error('Build.Wasm', 'Unable to find emcc. Try re-building with --clean-install flag.');
-    process.exit(2);
-  }
-  emcc = path.join(DEPS_EMSDK_EMSCRIPTEN, emcc);
-  npmlog.info('Build.Wasm', `(2/4) Find path to emcc... DONE, emcc: ${emcc}`);
-
-  // Step 3: Prepare build config
-  npmlog.info('Build.Wasm', '(3/4) Preparing build config...');
-  // tslint:disable-next-line:non-literal-require
-  const wasmBuildConfig = require(SRC_WASM_BUILD_CONFIG);
-  const exportedFunctions = wasmBuildConfig.exported_functions as string[];
-  const srcPatterns = wasmBuildConfig.src as string[];
-  if (exportedFunctions.length === 0) {
-    npmlog.error('Build.Wasm', `No exported functions specified in the file: ${SRC_WASM_BUILD_CONFIG}`);
+  if (!fs.existsSync(OUT_WASM)) {
+    npmlog.error('Build.Wasm', 'Please make sure onnxruntime_wasm.wasm is built and exists in /dist/');
```

Comment:
Hi, OUT_WASM needs to be changed to "onnxruntime_wasm_threads.wasm" for multi-threading WebAssembly. Please also consider this in the PR. Thanks.